source/templates/writing-helpers.md

```diff
@@ -59,11 +59,11 @@ You may also find yourself rendering your view classes in multiple
 places using the `{{view}}` helper. In this case, you can save yourself
 some typing by registering a custom view helper.
 
-For example, let’s say you have a view called `App.CalendarView`.
+For example, let’s say you have a view called `CalendarView`.
```

Comment:
Maybe I'm missing something here. In this case, the filename of the view could be `views/calendar-view.js`, but wouldn't the exported view by named `CalendarView` or are you suggesting that just 'Calendar' would suffice?